diff -rupN src_ori/javascript/idpselect.js src/javascript/idpselect.js
--- src_ori/javascript/idpselect.js	2023-06-06 08:49:16.207035294 +0200
+++ src/javascript/idpselect.js	2023-06-06 09:06:20.177194455 +0200
@@ -471,6 +471,9 @@ function IdPSelectUI() {
             if (regexp.test(testString)) {
                 return true;
             }
+            if (testString.startsWith(regexp[i])) {
+                return true;
+            }
         }
         return false;
     };
@@ -1615,4 +1618,3 @@ function IdPSelectUI() {
     };
 }
 
-(new IdPSelectUI()).draw(new IdPSelectUIParms());
diff -rupN src_ori/resources/index.html src/resources/index.html
--- src_ori/resources/index.html	2023-06-06 08:49:16.209035309 +0200
+++ src/resources/index.html	2023-06-06 09:06:17.618173038 +0200
@@ -14,7 +14,10 @@
 
   <script src="idpselect.js" type="text/javascript" language="javascript"></script>
 
-
+  <script type="text/javascript" language="javascript">
+    (new IdPSelectUI()).draw(new IdPSelectUIParms());
+  </script>
+  
   <noscript>
     <!-- If you need to care about non javascript browsers you will need to 
          generate a hyperlink to a non-js DS.
